predicators/structs.py

```diff
@@ -301,6 +301,24 @@ def _negated_classifier(self, state: State,
         # Separate this into a named function for pickling reasons.
         return not self._classifier(state, objects)
 
+    def __lt__(self, other: Predicate) -> bool:
+        return str(self) < str(other)
+
+
+@dataclass(frozen=True, order=False, repr=False)
+class VLMPredicate(Predicate):
+    """Struct defining a predicate that calls a VLM as part of returning its
+    truth value.
+
+    NOTE: when instantiating a VLMPredicate, we typically pass in a 'Dummy'
```

Comment:
can we make a small utils function that creates VLMPredicates but doesn't require the user to manually specify the dummy classifier? `create_vlm_predicate(name, types, get_vlm_query_str) -> VLMPredicate`